christfellowshipchurch/web-app-v2
# Horizontal Card List Update for Mobile View

### About
This PR implements an update to the Horizontal Card List Feature that changes this feature from having buttons to and a carousel to just scrolling when there is an overflow **Only on Mobile View**. 

**THIS COULD AFFECT OTHER PAGES OR COMPONENTS THAT USE HIGHLIGHT CARD - TEST THOROUGHLY**

### Test Instructions
1. Go to the live sisterhood-shownotes-plus page, check how the Horizontal Card List Feature looks and then compare with the testing link. [live](https://www.christfellowship.church/studies/sisterhood-shownotes-plus), [testing link](https://web-app-v2-git-horizontal-card-53f06c-christ-fellowship-church.vercel.app/studies/sisterhood-shownotes-plus)

2. Check that cards for **ALL** views (mobile, iPad, and desktop) in various pages in the testing link.

3. Test Prayer Cards List.

### Screenshots
|Before|After|
|--|--|
|<img width="1077" alt="Screenshot 2023-09-26 at 3 13 10 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/94265294/8743f8b4-fd09-42be-b35c-f0597d6e12fb">|<img width="1076" alt="Screenshot 2023-09-26 at 3 13 33 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/94265294/ff5fd666-adca-451d-be02-f3546bfc5804">|

### Closes Tickets
[CFDP-2913](https://christfellowshipchurch.atlassian.net/browse/CFDP-2913)

[CFDP-2913]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2913?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
